src/Moryx/Configuration/ConfigManagerExtensions.cs

```diff
@@ -0,0 +1,102 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Moryx.Configuration
+{
+    public static class ConfigManagerExtensions
+    {
+        /// <summary>
+         /// Get typed configuration. Will use cached object if available
+         /// </summary>
+         /// <typeparam name="T">Type of config object</typeparam>
+         /// <returns>Configuration object</returns>
+        public static T GetConfiguration<T>(this IConfigManager configManager) 
+            where T : class, IConfig, new()
+        {
+            var configType = typeof(T);
+            return (T)configManager.GetConfiguration(configType, configType.FullName, false);
+        }
+
+        /// <summary>
+        /// Get typed configuration. Will use cached object if available
+        /// </summary>
+        /// <typeparam name="T">Type of config object</typeparam>
+        /// <returns>Configuration object</returns>
+        public static T GetConfiguration<T>(this IConfigManager configManager, string name)
+            where T : class, IConfig, new()
+        {
+            var configType = typeof(T);
+            return (T)configManager.GetConfiguration(configType, name, false);
+        }
+
+        /// <summary>
+        /// Get typed configuration. Also specifies behaviour for implementations with internal cache
+        /// </summary>
+        /// <typeparam name="T">Type of config object</typeparam>
+        /// <param name="getCopy"><value>True</value>Create new instance. <value>False</value>Get from cache if possible</param>
+        /// <returns>Configuration object</returns>
+        public static T GetConfiguration<T>(this IConfigManager configManager, bool getCopy)
+            where T : class, IConfig, new()
+        {
+            var configType = typeof(T);
+            return (T)configManager.GetConfiguration(configType, configType.FullName, getCopy);
+        }
+
+        /// <summary>
+        /// Get typed configuration. Also specifies behaviour for implementations with internal cache
+        /// </summary>
+        /// <typeparam name="T">Type of config object</typeparam>
+        /// <param name="getCopy"><value>True</value>Create new instance. <value>False</value>Get from cache if possible</param>
+        /// <param name="name">Will lookup the config by the given name</param>
+        /// <returns>Configuration object</returns>
+        public static T GetConfiguration<T>(this IConfigManager configManager, bool getCopy, string name)
+            where T : class, IConfig, new()
+        {
+            return (T)configManager.GetConfiguration(typeof(T), name, getCopy);
+        }
+
+        /// <summary>
+        /// Get typed configuration. Also specifies behaviour for implementations with internal cache
+        /// </summary>
+        /// <typeparam name="T">Type of config object</typeparam>
```

Comment:
Copy paste error for name and type🙈

Reply:
I don't see the error

Reply:
![image](https://user-images.githubusercontent.com/37485711/194217815-84dbe27a-e4f6-4a98-bcdb-93e1020b7328.png)
😅
Please also check the other xml comments in this file based on the remarks from github 😊

Reply:
Got it